add highlight-self

bmwang opened the pull request:
i like your overlay, and the only issue i was having was being able to find myself quickly, at a glance.

i made a small mod to highlight yourself

![image](https://user-images.githubusercontent.com/868700/61278995-1dcf2f00-a76a-11e9-8656-0a3dd00bcdf2.png)

bmwang commented:
i've also made a second commit to add a group-by-4, as my friend requested it when i showed him your overlay, however i had to fix some z-indexing issues with the c-details pane so that they overlay correctly. the changes are a bit deep so no worries if you don't want to merge it:

![image](https://user-images.githubusercontent.com/868700/61285304-84a61580-a775-11e9-9883-5f667dff139c.png)

hibiyasleep commented:
(please note, I'm not familiar with english writing, words may seems rude but that's definitely not intended.)

* Those `-ㅁ-` only indicates that's yourself? this style of highlighting seems nice, but it would cause distraction by two presentive lines (DPS/Crit) that already exists.
*  Group by-4 also looks awesome, but what happens if user jumps into 24-player duty, FATE or Eureka? How about make this option configurable (Group by _n_), or we should limit player region's total height.

bmwang commented:
no worries, thanks for taking a look.

i cleaned it up a bit, fixed the element 2: none, and changed the lines to gradients. group by 4 is now 2 rows
![image](https://user-images.githubusercontent.com/868700/61368331-7d990900-a842-11e9-957a-d389b97fbee7.png)

![image](https://user-images.githubusercontent.com/868700/61368222-43c80280-a842-11e9-8679-904b3d538f68.png)

![image](https://user-images.githubusercontent.com/868700/61368411-a4efd600-a842-11e9-89aa-987e25b8a9a9.png)
